jormungandr-lib/src/interfaces/value.rs

```diff
@@ -59,7 +59,13 @@ impl AsRef<value::Value> for Value {
 
 impl From<value::Value> for Value {
     fn from(v: value::Value) -> Self {
-        Value(v)
+        Self(v)
+    }
+}
+
+impl From<&value::Value> for Value {
```

Comment:
this is redundant and actually inefficient

Reply:
I have added to be respectful how it is done with the `AccountState` in this place https://github.com/input-output-hk/jormungandr/blob/9b6db4f32577905f5d14fc067068547fc5dfb490/jormungandr/src/rest/v0/logic.rs#L119 and https://github.com/input-output-hk/jormungandr/blob/33993d0642a3494158297492d75dd0f4ece37da1/jormungandr-lib/src/interfaces/account_state.rs#L162. But if you think that it better to explicitly convert types I can easily change it.

Reply:
I mean that we already have a `From<value::Value> for Value` implementation, which makes this not needed. In addition, this involves an unnecessary clone where `value::Value` is `Copy`, which may have performance impacts.